### PR TITLE
fix(Core/Quest): They're Alive! Maybe...

### PR DESCRIPTION
**Pull request by Helias**
## Changes Proposed:
Fix the spawn of the expedition researcher.
Removed the ID **17680** because it's the cocoon NPC (the one that you kill to let free the expedition researcher).

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4009

## Tests Performed:
- tested in-game

## How to Test the Changes:
- .go -1541 -10772 53 530
- .quest add 9670
- attack the cocoons to open them
- no expedition researcher will appear

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp [hunk 1]
```diff
@@ -51,17 +51,20 @@ class npc_webbed_creature : public CreatureScript
             switch (urand(0, 2))
             {
                 case 0:
-                    if (Player* player = killer->ToPlayer())
+                    if (Player* player = killer->ToPlayer()) {
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp:54]:
```suggestion
                    if (Player* player = killer->ToPlayer()) 
                    {
```

File: src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp [hunk 2]
```diff
@@ -51,17 +51,20 @@ class npc_webbed_creature : public CreatureScript
...
+                    spawnCreatureID = NPC_EXPEDITION_RESEARCHER;
                     break;
                 case 1:
                 case 2:
                     spawnCreatureID = possibleSpawns[urand(0, 30)];
                     break;
             }
 
-            if (spawnCreatureID)
+            if (spawnCreatureID) {
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp:65]:
```suggestion
            if (spawnCreatureID) 
            {
```

**Comment by Helias** [comment]
ty @Kitzunu

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by wizzymore** [comment]
Working perfect

**Comment by Helias** [comment]
Thanks for the test and the feedback, feel open to free a new PR for these SAI improvements :wink:

**Comment by Kitzunu** [comment]
Thanks=)

**Pull request merged**